mlbench/master/api/models/kubepod.py

```diff
@@ -4,11 +4,11 @@
 
 
 class KubePod(models.Model):
-    name = models.CharField(max_length=50)
-    labels = models.CharField(max_length=255)
+    name = models.CharField(max_length=1000)
+    labels = models.CharField(max_length=1000)
     phase = models.CharField(max_length=20)
     ip = models.CharField(max_length=15)
-    node_name = models.CharField(max_length=100)
+    node_name = models.CharField(max_length=1000)
```

Comment:
You can't just change the DB fields in code, you also need to create Django migrations `python manage.py makemigrations`

But I can do this later this afternoon.